packages/frontend/src/pages/SavedSearches/SavedSearches.tsx

```diff
@@ -53,19 +54,22 @@ const autoFormatRelativeTime = (date: Date, locale = 'nb-NO'): string => {
   }
 };
 
-function getMostRecentSearchDate(searches: SavedSearchDTO[]): Date | null {
-  if (searches.length === 0) {
+function getMostRecentSearchDate(data: SavedSearchesFieldsFragment[]): Date | null {
+  try {
+    if (!data?.length) {
+      return null;
+    }
+    const timestamp = data?.reduce((latest, search) => {
```

Comment:
Syns denne koden er vanskelig å forstå, tenker intuitivt at slice().sort( // sortere på updatedAt)[0] er lettere å forstå for å slippe parseInt med `radix` og `reduce`.

Reply:
Sliter fortsatt med å forstå denne